packages/coding-standard/tests/Rules/TooDeepNewClassNestingRule/Fixture/AllowedDeepNestingNewClass.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Tests\Rules\TooLongVariableRule\Fixture;
+
+new A(
+    new B(
+        new C(),
+        new D(),
+        new E(),
+    )
+);
```

Comment:
I'm curious, this should be reported too.
Here is 5 `new ()` nestings, and 3 are allowed, right?

Reply:
It is not, because the new is not deep inside, but in the same level.

Reply:
Do you want same level `new` counted as well? If yes, then the following is not allowed as well?

```php
new A(
    new B,
    new C,
    new D
);
```

Reply:
Yes, those should be counted too.

Yes, this is not allowed

Reply:
implemented.